### DISCUSSION TITLE
ImageMagickObject.dll

### DISCUSSION THREAD

**Discussion by jmdl1983** [ImageMagick/ImageMagick#5498] [Help] [answered]
Years back I installed an older version ImageMagick-6.8.2-Q16 and it came with the file ImageMagickObject.dll included.

However with newer versions this does not seem to be the case.

According to documentation at: [https://imagemagick.org/script/ImageMagickObject.php](https://imagemagick.org/script/ImageMagickObject.php) , it should be included with the installer.

Does anyone know where to locate this file, or is the documentation out of date?

**Comment by fmw42** [comment]
Does this help?  https://github.com/ImageMagick/ImageMagick/discussions/3098

**Comment by dlemstra** [comment, marked as answer]
We no longer support ImageMagickObject.dll (see #4820). We will make sure that the page that you mention will be removed from our website.

**Reply by jmdl1983** [reply]
> We no longer support ImageMagickObject.dll (see #4820). We will make sure that the page that you mention will be removed from our website.

In reading this post it appears as the DLL last was included in version 7.1.0-19.  I cannot however find this download anywhere.  Do you host old versions anywhere, or does anyone have a copy of the last DLL version?
